omnibus/files/private-chef-cookbooks/private-chef/recipes/default.rb

```diff
@@ -230,36 +238,41 @@
   "nginx",
   "keepalived"
 ].each do |service|
-  if node["private_chef"][service]["enable"]
-    include_recipe "private-chef::#{service}"
+  if node["private_chef"][service]["external"]
+    begin
+      # Perform any necessary configuration of the external service:
+      include_recipe "private-chef::#{service}-external"
```

Comment:
@marcparadise I think it is probably best to do a few more services and see how it pans out.  In some cases it feels like the only difference between "external" and "internal" ought to be that we need to install the service for "internal".  In that world we could have an "installation" cookbook and a "configure/bootstrap" cookbook for each service.  When external was set, we wouldn't run the installation cookbook.

That said, we can see how it goes, this comment shouldn't block merging this.